Infrastucture docs have moved

ljones140 opened the pull request:
Removes infrastructure documentation as that has been moved to the new private repo https://github.com/clearlydefined/infrastructure

elrayle reviewed (commented):
I did not do a full review.  I did have a suggestion for the overview.md file.  The suggestion described might apply to other files as well.

ljones140 commented:
@elrayle and @qtomlinson I've added the env var configurations back.
I've put them in separate files to make reading easier.
And moved anything deployment focussed to the operations repo

qtomlinson reviewed (commented):
Thank you for separating the configs for the three services.  It is much clearer!
